### DISCUSSION TITLE
Assign Conversation to Agent via API

### DISCUSSION THREAD

**Discussion by Stephen-Lobo** [chatwoot/chatwoot#2674] [Q&A]
How to assign conversation to the particular Agent through API ?

Most of the time, we want to assign new conversation to the particular agent depending on timing / department. I think at present, assignment is done via smart algorithm depending who is ONLINE. But this is causing issues at our end.

**Comment by sojan-official** [comment]
You can use https://www.chatwoot.com/developers/api#operation/assign-a-conversation

I have also added an issue for further enhancements https://github.com/chatwoot/chatwoot/issues/2679
